pallets/runtime/common/src/runtime.rs

```diff
@@ -86,15 +79,36 @@ macro_rules! misc_pallet_impls {
             ///
             /// This type is being generated by `construct_runtime!`.
             type PalletInfo = PalletInfo;
+            /// The data to be stored in an account.
+            type AccountData = pallet_balances::AccountData<Balance>;
             /// What to do if a new account is created.
             type OnNewAccount = ();
             /// What to do if an account is fully reaped from the system.
             type OnKilledAccount = ();
-            /// The data to be stored in an account.
-            type AccountData = pallet_balances::AccountData;
             type SystemWeightInfo = polymesh_weights::frame_system::SubstrateWeight;
+            type ExtensionsWeightInfo = ();
+            /// The designated SS85 prefix of this chain.
+            ///
+            /// This replaces the "ss58Format" property declared in the chain spec. Reason is
+            /// that the runtime should know about the prefix in order to make use of it as
+            /// an identifier of the chain.
+            type SS58Prefix = SS58Prefix;
+            /// What to do if the runtime wants to change the code to something new.
+		    ///
+		    /// The default (`()`) implementation is responsible for setting the correct storage
+		    /// entry and emitting corresponding event and log item. (see
+		    /// [`Pallet::update_code_in_storage`]).
+		    /// It's unlikely that this needs to be customized, unless you are writing a parachain using
+		    /// `Cumulus`, where the actual code change is deferred.
             type OnSetCode = ();
+            /// The maximum number of consumers allowed on a single account.
             type MaxConsumers = frame_support::traits::ConstU32<16>;
+            /// The set code logic, just the default since we're not a parachain.
+            type SingleBlockMigrations = ();
+            type MultiBlockMigrator = ();
```

Comment:
I think some of the Substrate pallets we use (like `pallet_contracts`) use the MultiBlockMigrator, which means we will need to include the `pallet_migrations` into our runtime.